ansible/aix61-standalone/manualBootstrap.md

```diff
@@ -9,19 +9,136 @@ modules will be installed.
 
-               6.1-1          6.1-2     7.2-2
-	/opt: 3670016 x 512, 2228224  31457280
-	/:    3538944 x 512, 3407872   4194304
+```bash
+chfs -a size=23000000 /home
+```
+
+## Edit SSH config
+
+Add the following two lines to `/etc/ssh/sshd_config`:
+
+```
+PasswordAuthentication no
+ChallengeResponseAuthentication no
+```
+
+## Remove failedlogin file
+
+If /etc/security/failedlogin is growing without bounds on AIX6.1, remove
+it with a cron job, use `crontab -e`, and add one line:
+```sh
+0 12 * * * /usr/bin/rm -f /etc/security/failedlogin
+```
+
+## Fix "Missing" shared objects
+
+On the 7.1 machines we were facing this issues when yum installed packages were not able to find some of the shared objects they needed
+
+```sh
+bash-5.0$ gmake -v
+exec(): 0509-036 Cannot load program gmake because of the following errors:
+        0509-022 Cannot load module /opt/freeware/lib/libintl.a(libintl.so.8).
+        0509-150   Dependent module /usr/lib/libiconv.a(libiconv.so.2) could not be loaded.
+        0509-152   Member libiconv.so.2 is not found in archive
+        0509-022 Cannot load module make_64.
+        0509-150   Dependent module /opt/freeware/lib/libintl.a(libintl.so.8) could not be loaded.
+        0509-022 Cannot load module .
+```
+
+The fix is as following:
+
+```sh
+sudo rm /usr/lib/libiconv.a && sudo ln -s /opt/freeware/bin/libiconv.a /usr/lib
+```
+
+# AIX 7.2 Install
+
+Most packages should be installed via ansible.
+
+If there are any missing they should be installed via yum
+
+What you do need to install manually is **gcc** and **ccache**
+
+## gcc 6.3.x on AIX 7.2
+
+```bash
+mkdir -p /opt/gcc-6.3 && cd /opt/gcc-6.3
+curl -L https://ci.nodejs.org/downloads/aix/gcc-6.3-aix7.2.ppc.tar.gz | /opt/freeware/bin/tar -xzf -
+```
+
+## ccache 3.7.4 on AIX 7.2
+
+```bash
+mkdir -p /opt/ccache-3.7.4 && cd /opt/ccache-3.7.4
+curl -L https://ci.nodejs.org/downloads/aix/ccache-3.7.4.aix7.2.ppc.tar.gz | /opt/freeware/bin/tar -xzf -
+```
+## Enable the AHA fs
+
+For AIX 7.2 and 6.1, needed for the file watcher unit tests.
+
+Add the following to /etc/filesystems:
+
+```
+/aha:
+        dev             = /aha
+        vfs             = ahafs
+        mount           = true
+        vol             = /aha
+```
+
+and then:
+
+```bash
+mkdir /aha
+mount /aha
+```
+
+## Install XL compilers
+
+1. Download 16.1.0 packages from: https://testcase.boulder.ibm.com (username:
+   xlcomp4, password: ask @mhdawson)
+2. scp them to target:/opt/ibm-xlc
+3. on target:
+```bash
+cd /opt/ibm-xlc
+uncompress 16.1.0.3-IBM-xlCcmp-AIX-FP003.tar.Z
+uncompress IBM_XL_C_CPP_V16.1.0.0_AIX.tar.Z
+installp -aXYgd ./usr/sys/inst.images -e /tmp/install.log all
+inutoc
+installp -aXgd ./ -e /tmp/install.log all
+```
+4. Find compilers in `/opt/IBM/xl[cC]/16.1.0/bin/`
+
+
+
+#### tap2junit
+
+Should be installed via ansible but if needed to be done manually use the following:
```

Comment:
I think it was done by ansible on aix7, wasn't it?

Reply:
yes think so too

Reply:
so the whole section can be removed.